src/main/java/org/tarantool/MsgPackLite.java

```diff
@@ -72,13 +72,6 @@
 
     public void pack(Object item, OutputStream os) throws IOException {
         DataOutputStream out = new DataOutputStream(os);
-        if (item instanceof Callable) {
```

Comment:
There is no asynchronous case in the code I suggest to remove and thus no asynchronous workaround is needed.

This code violates single responsibility principle (mixing serialization and execution) and hence the disturbance.